### ISSUE TITLE
rem/em for rowHeight?

### ISSUE THREAD

**Issue by FateRiddle** [bvaughn/react-virtualized#707]
# Are you making a feature request?
I was writing the whole css in `rem`. (or trying css framework like `tachyons` that layout using `rem`).
Then I find the `List`/`Table` component only accept `px`. `<AutoSizer/>` helps with `height` amd `width`, but `rowHeight` seems to only accept number which will translate into `px`.
Can there be a way to flexibly setup rowHeight?(be it px,rem,em,vh) I certainly see use cases for react-virtualized to integrate into some projects more smoothly.

**Comment by bvaughn** [comment]
You might look into using a library like [tysonmatanich/getEmPixels](https://github.com/tysonmatanich/getEmPixels) to convert rem to px (eg `getEmPixels()` returns rem for the document). I don't think this functionality belongs in react-virtualized though because it's not really related to windowing.

**Comment by FateRiddle** [comment]
@bvaughn But it relates to sizing. And unit of measure is a part of sizing.

**Comment by bvaughn** [comment]
By that logic, any react component that has anything to do with sizing/measurements would need to support em/rem and percentage based units. 😄

This library is about windowing. It's necessary to deal with sizing/dimensions in order to implement window, but it's not the primary focus of the library and there are other NPM modules that are focused on that (like the one I linked to). It's better for maintainability (and my sanity) to keep this library focused. 😄 Shouldn't be too hard for you to use the lib I linked to, right?
